src/components/VideosGrid/VideoGridItem/VideoGridItem.component.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import {motion} from 'framer-motion';
+
+const CardContainer = styled(motion.div)`
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    margin: 10px;
+    max-width: 340px;
+    min-width: 340px;
+    max-height: 280px;
+    min-height: 280px;
+    text-align: left;
+`;
+
+const VideoThumbnail = styled.img`
+    max-height: 180px;
+    object-fit: cover;
+`;
+
+const VideoInfo = styled.div`
```

Comment:
As I mentioned you can move the styled components to their own files, when you have a lot of styled components it is easier to understand what's going when the code is ordered in a logical way, if there is a file that only contains the styles when someone needs to know something about styles just go to that file, it's not a rule, it depends on your preference or your team's preferences